### PR TITLE
fix hp a11y

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

**Comment by fiji-flo** [comment]
This just make the search form / input ids unique and adds a title to the mozilla svg

File: client/src/ui/organisms/top-navigation-main/index.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export const TopNavigationMain = ({ isOpenOnMobile }) => {
       <MainMenu isOpenOnMobile={isOpenOnMobile} />
 
       <Search
+        id="top-nav-search"
```
Comment (schalkneethling) [inline client/src/ui/organisms/top-navigation-main/index.tsx:35]: The skip to search link in the ally nav uses `#main-q`

Comment (schalkneethling) [reply client/src/ui/organisms/top-navigation-main/index.tsx:35]: So, we either need to update the a11y nav link or use the same id here.

Comment (fiji-flo) [reply client/src/ui/organisms/top-navigation-main/index.tsx:35]: 👍  good catch

File: client/src/assets/moz-logo.svg [hunk 2]
```diff
@@ -1 +1,5 @@
...
+<svg width="112" height="32" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (caugner) [inline client/src/assets/moz-logo.svg:1]: FWIW This change seems unrelated.

File: client/src/ui/organisms/top-navigation-main/index.tsx (hunk 1 above)
Comment (fiji-flo) [reply client/src/ui/organisms/top-navigation-main/index.tsx:35]: 🙈 that link is broken already…

File: client/src/assets/moz-logo.svg (hunk 2 above)
Comment (fiji-flo) [reply client/src/assets/moz-logo.svg:1]: I added the `<title>` for accessibility and formatted the file.
